Add F3 to halloffame and README

error401de opened the pull request:
Dependent on on https://github.com/geokrety/GeoKrety-Static/pull/7
Resolves https://github.com/geokrety/geokrety-website/issues/502

kumy commented:
Thanks for your contribution. May you please:
* fix the travis issue (seems to be only a probleme of file formating)
* Use `svg` file
* And if you wish create another PR for our current dev branch called `feature/new-theme` - be carreful, it's a complete website rewrite, so this PR cannot be pushed to `feature/new-theme` as is ;)

error401de commented:
Hi @kumy 
I fixed the issue and switched to svg files.
Mentioned issue at travis is gone now, anyhow there's a new one at "quatests" i'm not sure about what it means though.

boly38 commented on README.md:
```diff
@@ -10,4 +10,6 @@ Please contact us directly for any security issue contact@geokretymap.org
-
+# Credits
+![powered by F3](https://cdn.geokrety.org/images/logos/f3.svg)\
+Powered by [Fat-Free Framework](https://fatfreeframework.com)
```
I think F3 is introduced with **feature/new-theme** branch, so what about making this PR for **feature/new-theme** branch as target (instead of geokrety:master)

but we dont have credits in new-theme right now @kumy ?

kumy commented:
Ho good point @boly38, totally forgot that. And yes, credits are implemented in new theme.

kumy commented:
@error401de qatests error may be ignored. But @boly38 is right. This PR should be only for branch `feature/new-theme`.

boly38 commented on website/templates/konfig-credits.php:
```diff
@@ -124,3 +124,10 @@
     'link' => 'https://www.google.com/recaptcha/',
     'desc' => 'Secure captcha (stop bots)',
 ];
+$config['gk_credits'][] = [
```
if you target new-theme, then new related file is `website\app\GeoKrety\Controller\HallOfFame.php`

error401de commented:
As discussed I created another PR https://github.com/geokrety/geokrety-website/pull/517 :)
